commit 0c9930081862d95fd36e241b7350e83b2f514618
Author: Sam Thursfield <sam@afuera.me.uk>
Date:   Mon Jul 20 11:43:22 2020 +0200

    flatpak: Use pulseaudio via ALSA as default audio backend
    
    We were previously accessing pulseaudio via OSS, but this worked very
    badly.
    
    By using the 'pulse' audio device we can work against pulseaudio by
    default, and direct hardware access is still available for advanced
    users who are willing to give Purr Data exclusive soundcard access.
    
    Fixes https://github.com/flathub/net.purrdata.PurrData/issues/10

diff --git a/packages/linux_flatpak/pd-wrapper.sh b/packages/linux_flatpak/pd-wrapper.sh
index 541a195e..e056fa6c 100644
--- a/packages/linux_flatpak/pd-wrapper.sh
+++ b/packages/linux_flatpak/pd-wrapper.sh
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-padsp -n "Pure Data (Flatpak)" pd-l2ork -oss $@
+pd-l2ork $@
